templates/support/livepatch.html

```diff
@@ -0,0 +1,160 @@
+{% block meta_description %}Automatic updates for Ubuntu 14.04, 16.04, and 18.04 LTS, reducing downtime whilst keeping on top of compliance and security.{% endblock %}
+{% block meta_copydoc %}https://docs.google.com/document/d/1ObUSvkDcWOe01pmhh_Aryqxs87D8X2khbJbn3RgqZE0/edit{% endblock meta_copydoc %}
+
+{% block content %}
+<section class="p-strip is-deep">
+  <div class="row">
+    <div class="col-6">
+      <h1 style="font-weight: 100;">Canonical Livepatch Service</h1>
+      <h2 class="p-heading--four">Apply critical kernel patches without rebooting.</h2>
+      <ul class="p-list">
+        <li class="p-list__item is-ticked">Fixes are applied automatically, without restarting your system</li>
+        <li class="p-list__item is-ticked">Reduces downtime, keeping your Ubuntu LTS systems secure and compliant</li>
+        <li class="p-list__item is-ticked">Included as part of all Ubuntu Advantage support packages</li>
+      </ul>
+    </div>
+    <div class="col-6">
+      <div class="u-embedded-media">
+        <iframe class="u-embedded-media__element" width="772" height="426" src="https://player.vimeo.com/video/225445183" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen></iframe>
+      </div>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-12">
+      <hr style="margin: 5rem 0;" />
+    </div>
+  </div>
+  <div class="row">
+    <div class="col-10">
+      <h2>Subscribe to Livepatch</h2>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://auth.livepatch.canonical.com/"><span class="p-link--external">Free for up to 3 machines</span></a>
+        <span style="padding-left: 2rem;">All you need is an Ubuntu One account.</span>
+      </p>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://aws.amazon.com/marketplace/seller-profile?id=565feec9-3d43-413e-9760-c651546613f2"><span class="p-link--external">AWS Marketplace</span></a> <span style="padding-left: 2rem;">Part of Ubuntu Advantage for AWS.</span></p>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://azuremarketplace.microsoft.com/en-us/marketplace/apps/Canonical.UbuntuAdvantageSupport?src=blogazure&tab=Overview"><span class="p-link--external">Azure Marketplace</span></a>
+        <span style="padding-left: 2rem;">Part of Ubuntu Advantage for Azure.</span>
+      </p>
+      <p class="u-no-max-width">
+        <a class="p-button--positive p-button--wide"  href="https://buy.ubuntu.com"><span class="p-link--external">Buy direct</span></a>
+        <span style="padding-left: 2rem;">Part of Ubuntu Advantage for desktops or datacenters.</span>
+      </p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row">
+    <h2 class="p-muted-heading u-align-text--center">Used by</h2>
+    <div class="col-12 prefix-1">
+      <ul class="row">
```

Comment:
I still feel the `p-inline-images` is a better pattern here. It aligns logos nicely in the centre and does not require nested grids.

Reply:
the issue is that the logos are _tiny_ if we add them, they have a lot of whitespace.